### ISSUE TITLE
Remember form fields

### ISSUE THREAD

**Issue by Htbaa** [elm-lang/cookie#8]
I was considering using Elm to redo a contactform for putting in orders, mainly because based on user input calculations have to be made. This form also has a 'remember me' checkbox so clients can come back later and when they place another order their contact information is already filled in. This is not something that's being stored inside a database, but inside of a cookie.

Currently that form (plain old JavaScript with a bit of MooTools) stores the inputfields inside a cookie right before submitting the form to a mailscript.

Basically being able to write and read cookies like MooTools Utilities/Cookies (https://mootools.net/core/docs/1.6.0/Utilities/Cookie) would be a nice thing. Or is this already possible?
